Teapot/Teapot+BasicAuth.swift

```diff
@@ -8,7 +8,7 @@ extension String {
 
 public extension Teapot {
     /// The basic authentication header key value. Use this as the key in your headerFields dictionary.
-    public var basicAuthenticationHeaderKey: String {
+    var basicAuthenticationHeaderKey: String {
```

Comment:
why does this no longer need to be public, because the extension itself is declared as public?

Reply:
Precisely. Now they 'inherit' the access level.